src/main/java/uk/gov/justice/digital/nomis/controller/SentenceCalculationsController.java

```diff
@@ -48,9 +49,10 @@ public SentenceCalculationsController(SentenceCalculationsService sentenceCalcul
                     value = "Number of records per page.")})
     public PagedResources<Resource<SentenceCalculation>> getSentenceCalculations(
             final @ApiIgnore Pageable pageable,
+            @RequestParam(value = "bookingId", required = false) Set<Long> bookingId,
             final PagedResourcesAssembler<SentenceCalculation> assembler) {
 
-        Page<SentenceCalculation> sentenceCalculations = sentenceCalculationsService.getSentenceCalculations(pageable);
+        Page<SentenceCalculation> sentenceCalculations = sentenceCalculationsService.getSentenceCalculations(pageable, bookingId);
```

Comment:
@steverendell param is not showing up in Swagger - think you need a API swagger annotation here